packages/interfaces/src/filter.ts

```diff
@@ -25,7 +25,7 @@ export interface IFilterSubscription {
 export type IFilter = IReceiver &
   IBaseProtocol & {
     createSubscription(
-      pubsubTopicShardInfo?: SingleShardInfo,
+      pubsubTopicShardInfo?: SingleShardInfo | string,
```

Comment:
let's maintain using `PubsubTopic` as the type instead of `string`

Reply:
fixed in latest commit